screens/join.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/style.css" />
+    <link rel="stylesheet" href="../css/pages/login/login.css" />
+    <title>판다마켓 - join</title>
+  </head>
+  <body>
+    <div class="total-container">
+      <main class="loginContainer">
+        <a href="../index.html" class="login-logo">
+          <img src="../img/logo.png" / >
+        </a>
+        <div class="login-form-container">
+          <form class="login-form">
+            <label for="login-email"> 이메일 </label>
+            <input
+              id="login-email"
+              type="email"
+              placeholder="이메일을 입력해주세요"
+              required
+            />
+            <label for="login-nickname"> 닉네임 </label>
+            <input
+              id="login-nickname"
+              type="text"
+              placeholder="닉네임을 입력해주세요"
+              required
+            />
+            <label for="login-password"> 비밀번호 </label>
+            <input
+              id="login-password"
+              type="password"
+              placeholder="비밀번호를 입력해주세요"
+              required
+            />
+            <label for="login-password2"> 비밀번호 확인</label>
+            <input
+              id="login-password2"
+              type="password"
+              placeholder="비밀번호를 다시 입력해주세요"
+              required
+            />
+            <input type="submit" value="회원가입" id="login-submit" />
```

Comment:
❗️ 수정요청
해당 요소는 form 을 제출하는 역할을 하므로 button 요소가 더 적절합니다~

```suggestion
            <button type="submit" id="login-submit">회원가입</button>
```